packages/flutter/test/services/text_input_test.dart

```diff
@@ -26,40 +27,53 @@ void main() {
       TextInput.setChannel(SystemChannels.textInput);
     });
 
-    test('text input client handler responds to reattach with setClient', () async {
```

Comment:
I've ran "flutter format" to format this file.

Reply:
The flutter framework does not use use formatter, see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#formatting. Can you please undo the unrelated format changes?

Reply:
Done.